devel/cram/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                cram
+version             0.7
+categories          devel python
+platforms           darwin
+license             GPL-2+
+maintainers         @SoapZA
+supported_archs     noarch
+installs_libs       no
+
+description         A simple testing framework for command line applications
+long_description    Cram is a functional testing framework for command line applications. \
+                    Cram tests look like snippets of interactive shell sessions. Cram runs \
+                    each command and compares the command output in the test with the \
+                    command's actual output.
+
+homepage            https://bitheap.org/cram/
+
+master_sites        pypi:c/${name}
+distname            ${name}-${version}
+
+checksums           rmd160  50e5389108496fec6904ccc3e911399b07b4d480 \
+                    sha256  7da7445af2ce15b90aad5ec4792f857cef5786d71f14377e9eb994d8b8337f2f \
+                    size    33527
+
+python.versions         27 34 35 36 37
```

Comment:
We'll start phasing out python 3.4 subports soon (potentially striving towards supporting just 3.6 and 3.7, not sure). Unless there's some strong reason for keeping support for 3.4, it's better to leave it out.